ldap/servers/slapd/back-ldbm/ldbm_entryrdn.c

```diff
@@ -1695,12 +1711,18 @@ _entryrdn_get_tombstone_elem(entryrdn_db_ctx_t *ctx,
             if (0 == strcmp(comma + 1, slapi_rdn_get_nrdn(srdn))) {
                 /* found and done */
                 _entryrdn_dup_rdn_elem((const void *)dataret.data, elem);
+                if (RDN_IS_REDIRECT(childelem)) {
+                    slapi_ch_free((void **)&childelem);
```

Comment:
this sounds suspicious because  childelem is an array elems ...

Reply:
> this sounds suspicious because childelem is an array elems ...

It fixes the leak.  Note childelem and dataret.data point to the same address.  I just chose to use childelem.